src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -294,19 +292,16 @@ function deleteQuestion(number) {
  * Allows users to cancel editing questions
  */
 function cancelEdit(number) {
-    if (number === -1) {
-        if (confirm('Are you sure you want to cancel adding this question?')) {
-            location.reload();
-        }
-    } else {
-        if (confirm('Are you sure you want to cancel your changes?')) {
-            location.reload();
-        }
+    var confirmationMsg = number === -1 ?
+            'Are you sure you want to cancel adding this question?' :
+            'Are you sure you want to cancel your changes?';
```

Comment:
Yes, that's the part I'm referring to. May be @MrFool can work on creating our own JS style guide or (preferred) switching to another JS guide that is consistent with our Java style guide). I'm hoping @yj-soh can similarly be in charge of the JSP style guide.